code/api/lecture_search_cli.py

```diff
@@ -2,11 +2,20 @@
 from pprint import pprint
 from lecture_search import engine
 
-max_results = 5  # make extensible?
-query_txt = str(input("Enter CS410 search query: "))
+query_txt = str(input("Enter search query: "))
+
+corpus = str(input("Enter corpus [lectures]: "))
```

Comment:
Would this work here?

```python
corpus = str(input("Enter corpus [lectures]: ")) or 'lectures'
```

Reply:
As above, you point out a good approach and I'll make this change to both parameters.